server/go.mod

```diff
@@ -7,6 +7,8 @@ require (
 	github.com/gin-gonic/gin v1.6.3
 	github.com/google/uuid v1.1.1
 	github.com/heroku/x v0.0.0-20171004170240-705849e307dd
+	github.com/jfeliu007/goplantuml v1.4.0 // indirect
 	github.com/patrickmn/go-cache v2.1.0+incompatible
+	github.com/spf13/afero v1.2.2 // indirect
```

Comment:
Just curious: What are these new deps?

Reply:
I didn't realize they got added, they can be taken out.  I was trying to learn a bit more about state design pattern so was using goplantuml to autogenerate uml diagram of drawydraw.  I think the other package got added at the same time, I will take out.